utilities/llmd_utils.py

```
@@ -0,0 +1,658 @@
+        cmd = self.generate_command(
+            model_name=model_name,
+            inference_input=inference_input,
+            use_default_query=use_default_query,
+            insecure=insecure,
+            token=token,
+        )
+
+        res, out, err = run_command(command=shlex.split(cmd), verify_stderr=False, check=False)
+        if res:
+            return out
+        raise ValueError(f"Inference failed with error: {err}\nOutput: {out}\nCommand: {cmd}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)